create_framework/http_kernel_httpkernelinterface.rst

```diff
@@ -64,6 +64,7 @@ PHP; it implements ``HttpKernelInterface`` and wraps another
         new HttpKernel\HttpCache\Store(__DIR__.'/../cache')
     );
 
+    /* $response variable used before and after this chapter -> keep it in this chapter for more global coherence? */
```

Comment:
Are you suggesting to rewrite like this: 

```
$response = $framework->handle($request);
$response->send();
```

Reply:
Yes, this is the way it is implemented in the chapters before and after this one, so might be nice to keep it this way here to improve global coherence of the code.

Reply:
I agree with that. 

Thank you